curriculum/Unit6/Lesson-601.md

```diff
@@ -338,7 +338,7 @@ Video
   <https://www.youtube.com/watch?v=Alv2ApK_jdo>
 
 - CSE 142, _Inheritance_ (23:28–35:06)<br>
```

Comment:
I've noticed the CSE 142 links are typically before the HomeworkBytes links. Is there a reason this is in a different order?